frontend/src/views/web/challenge-list.html

```diff
@@ -9,7 +9,8 @@
                         <img class="bg-img" ng-src="{{challenge.image}}">
                         <span class=" ev-card-title fs-14"><span><img ng-src="{{challenge.image}}" ></span>{{challenge.title}}</span>
                     </div>
-                    <div class="card-content">
+                    <div class="chip">test chip</div>
```

Comment:
@tanishqsandhu these chips need to be fetched from backend. Now that you have frontend ready you can add a new field in challenge model and pass it to frontend from get challenge api. Next step should be showing the title of the chip from data returned by the API

Reply:
Hey @tanishqsandhu, Can you incorporate this comment. Here are the steps that you will have to follow --
1. In the challenge controller, check if you are getting this `tags` field from the backend.
2. if yes, this `test chip` should be replaced by that.
3. if the field's value is none or blank, then it shouldn't appear.